.chloggen/sematextexporter-new-component.yaml

```
@@ -0,0 +1,21 @@
+# Use this changelog template to create an entry for release notes.
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: exporter/sematext
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add a new component for exporting metrics and logs to Sematext"
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [35540]
```

Perhaps better to use the newer issue number?

```suggestion
issues: [36465]
```